sycl/test/basic_tests/accessor/accessor_property_list_ct.cpp

```diff
@@ -0,0 +1,94 @@
+  {
+    // No properties
+    accessor_property_list PL{};
+    static_assert(!PL.has_property<property::no_alias>(), "Property is found");
+    static_assert(!PL.has_property<property::no_offset>(), "Property is found");
+  }
+
+  {
+    // Single CT property
+    accessor_property_list PL{no_alias};
+    static_assert(PL.has_property<property::no_alias>(), "Property not found");
+    static_assert(!PL.has_property<property::no_offset>(), "Property is found");
+  }
+
+  {
+    // Multiple CT properties
+    accessor_property_list PL{no_alias, no_offset};
+    static_assert(PL.has_property<property::no_alias>(), "Property not found");
+    static_assert(PL.has_property<property::no_offset>(), "Property not found");
+
+    static_assert(PL.get_property<property::no_alias>() == no_alias,
+                  "Properties are not equal");
+  }
+
+  {
+    int data[1] = {0};
+
+    sycl::buffer<int, 1> buf_data(data, sycl::range<1>(1),
+                                  {sycl::property::buffer::use_host_ptr()});
+    sycl::queue Queue;
+
+    Queue.submit([&](sycl::handler &cgh) {
+      accessor_property_list PL{no_alias};
+      sycl::accessor acc_1(buf_data, cgh, PL);
+      sycl::accessor acc_2(buf_data, cgh, sycl::range<1>(0), PL);
+      sycl::accessor acc_3(buf_data, cgh, sycl::range<1>(0), sycl::id<1>(1),
+                           PL);
+      sycl::accessor acc_4(buf_data, cgh, sycl::read_only, PL);
+      sycl::accessor acc_5(buf_data, cgh, sycl::range<1>(0), sycl::read_only,
+                           PL);
+      sycl::accessor acc_6(buf_data, cgh, sycl::range<1>(0), sycl::id<1>(0),
+                           sycl::read_only, PL);
+      sycl::accessor acc_7(buf_data, cgh, sycl::write_only, PL);
+      sycl::accessor acc_8(buf_data, cgh, sycl::range<1>(0), sycl::write_only,
+                           PL);
+      sycl::accessor acc_9(buf_data, cgh, sycl::range<1>(0), sycl::id<1>(0),
+                           sycl::write_only, PL);
+      cgh.single_task<class NullKernel>([]() {});
+    });
+    accessor_property_list PL{no_alias};
+    sycl::accessor acc_1(buf_data, PL);
+    sycl::accessor acc_2(buf_data, sycl::range<1>(0), PL);
+    sycl::accessor acc_3(buf_data, sycl::range<1>(0), sycl::id<1>(0), PL);
+    sycl::accessor acc_4(buf_data, sycl::read_only, PL);
+    sycl::accessor acc_5(buf_data, sycl::range<1>(0), sycl::read_only, PL);
+    sycl::accessor acc_6(buf_data, sycl::range<1>(0), sycl::id<1>(0),
+                         sycl::read_only, PL);
+    sycl::accessor acc_7(buf_data, sycl::write_only, PL);
+    sycl::accessor acc_8(buf_data, sycl::range<1>(0), sycl::write_only, PL);
+    sycl::accessor acc_9(buf_data, sycl::range<1>(0), sycl::id<1>(0),
+                         sycl::write_only, PL);
+  }
+
+  {
+    int data[1] = {0};
+
+    sycl::buffer<int, 1> buf_data(data, sycl::range<1>(1),
+                                  {sycl::property::buffer::use_host_ptr()});
+
+    accessor_property_list PL{sycl::noinit, no_alias};
```

Comment:
Is it possible to create `PL_1` from `PL`?
Would `{sycl::noinit, no_alias}` property be equal to `{no_alias, sycl::noinit}` property?

Reply:
I prefer to handle this comment later